incubator/openldap/templates/openldap-service.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Service spec for openldap
+#
```

Comment:
Most of these comments are useless and just state the obvious. I'd remove them.

Reply:
Removed comments.